### ISSUE TITLE
Failing Tests

### ISSUE THREAD

**Issue by zoffixznet** [tony-o/perl6-data-dump#17]
```
zoffix@ZofLap:/tmp/perl6-data-dump$ prove -e 'perl6 -Ilib' -vlr t
t/00-use.t .... 
1..1
ok 1 - can use
ok
t/01-hash.t ... 
1..1
ok 1 - 
ok
t/02-obj.t .... 
1..1
ok 1 - got expected data structure
ok
t/03-array.t .. 
1..1
[
  {
    hello => "world".Str,
  },
  {
    one => 2.Int,
  },
]
ok 1 - 
ok
t/04-gist.t ... 
1..3
not ok 1 - weird hash

# Failed test 'weird hash'
# at t/04-gist.t line 13
ok 2 - Match object
ok 3 - 
# Looks like you failed 1 test of 3
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 

Test Summary Report
-------------------
t/04-gist.t (Wstat: 256 Tests: 3 Failed: 1)
  Failed test:  1
  Non-zero exit status: 1
Files=5, Tests=7,  9 wallclock secs ( 0.09 usr  0.02 sys +  8.24 cusr  0.68 csys =  9.03 CPU)
Result: FAIL
zoffix@ZofLap:/tmp/perl6-data-dump$ 
```

**Comment by tony-o** [comment]
This should be corrected with a69dd41 - can you confirm please?

**Comment by zoffixznet** [comment]
Works fine now. Thanks.
